Bucketeer/Sources/Public/BKTError.swift

```diff
@@ -53,11 +59,17 @@ extension BKTError {
                     self = .badRequest(message: errorResponse?.error.message ?? "BadRequest error")
                 case 401:
                     self = .unauthorized(message: errorResponse?.error.message ?? "Unauthorized error")
+                case 403:
+                    self = .forbidden(message: errorResponse?.error.message ?? "Forbidden error")
                 case 404:
                     self = .featureNotFound(message: errorResponse?.error.message ?? "NotFound error")
                 case 405:
                     self = .invalidHttpMethod(message: errorResponse?.error.message ?? "MethodNotAllowed error")
-                case 500:
+                case 409:
+                    self = .clientClosed(message: errorResponse?.error.message ?? "Client Closed Request error")
+                case 503:
+                    self = .unavailable(message: errorResponse?.error.message ?? "Unavailable error")
+                case 500..<600:
```

Comment:
Please handle only the error 500.
By doing this is hard to debug when something happens between 500~599 errors.